# Overall successful exposure calibration
int8 SUCCESS = 1

# Failure to call service '/set_parameters'
int8 CAMERA_PARAM_SET_FAILURE = -1

# Requested camera parameter is out of bounds failure
int8 REQUESTED_PARAM_OUT_OF_BOUNDS_FAILURE = -2

# Failure to receive Image from callback
int8 IMAGE_NOT_RECEIVED_FAILURE = -3

# Requested camera blue value hasn't been achieved yet
int8 REQUESTED_CAMERA_BLUE_VALUE_NOT_MET = -4
